services/user.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> None:
+    user = User.objects.create_user(
+        username=username,
+        password=password
+    )
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:

```

Comment:
is not None

Reply:
Thx you! 
added contidion "is not None"